builtin/settingtypes.txt

```diff
@@ -2398,3 +2398,7 @@ reach (Reach) bool false
 airjump (AirJump) bool false
 
 spider (Spider) bool false
+
+node_light(Node stroker) bool false
```

Comment:
should be the same spelling as in cheat menu, so `node_light (NodeStroker) bool false`